### PR TITLE
Deathmatch : Final Destination

### PR DESCRIPTION
**Pull request by sylvia-from-fulp-station**
## About The Pull Request
My third and final deathmatch map, a competitive class based map. 
Four player fight to the death as NanoTrasen or the Syndicate's finest. 

Photoshop_oxIOnVeazn

## Why It's Good For The Game
- The only way to get this cool ~~power-gamer~~ stuff in any of the current DM maps is through a Mystery Box. This is fun and works on those maps. Arena Station, which uses classes over Mystery Boxes, is designed for many players; and said classes rely on item pickups. It's missing some of the more iconic 'characters' too.
- DM lobbies struggle to fill to the higher end of their cap without pre-round organization in OOC chat, (effectively killing the round on lowpop) and the current options for 2-4 players are lacking.
-  There's no map where you can have an energy sword fight.
- Adds a heap of classes that can be used in new maps.
/datum/outfit/deathmatch_loadout/dmcap,dmhos,dmtot,dmnukie,dmtider,dmabductor
## Changelog

:cl:
add: added dm_finaldestination.dmm and ~~7~~ **6** associated classes.

### CODE REVIEW

**Comment by DAKKA-WAAAGH** [comment]
I think the 'Disciple of Pete' class might be a little too much in terms of firepower.

Grenade spam can be fun but the other classes get out-shined pretty heavily; maybe give them something else other than the grenade belt? Cuban Pete was well known for voice activated bombs and other creative methods of causing mayhem.

The originally planned 4 classes and the extra 2 you got from ideas work really well though and it's very fun to play with and against them! The shrink ray is hilarious and gets a laugh from everybody lol.

Great work and looking forward to a merge!

**Comment by DAKKA-WAAAGH** [comment]
Additionally, I was told the Nukie didn't have the Nukie implant check in order to fire their primary gun. I have not tried it myself because I was too busy shrinking people.

**Comment by sylvia-from-fulp-station** [comment]
> Additionally, I was told the Nukie didn't have the Nukie implant check in order to fire their primary gun. I have not tried it myself because I was too busy shrinking people.

This was found pretty quickly, my mistake. I changed the Bulldog to the SMG without adding the /unrestricted version on the end. I fear using the implant check based versions and giving nukie the implant will lead to less engaging fights since you wont be able to use them as loot randomly.



> I think the 'Disciple of Pete' class might be a little too much in terms of firepower.

Hard agree (50~ tc of explosives btw) but if its too much the host can ask or just force change the class of people abusing it. All it lacks is precision.
I gave them a bag with remote signallers so hopefully people will spend time coming up with creative traps instead of, Belt bind, Activate in hand bind, Throw , repeat.

**Comment by sylvia-from-fulp-station** [comment]
Photoshop_QT8GYEc1i8
three classes added

File: fulp_modules/mapping/deathmatch/code/deathmatch_loadouts.dm
Comment (sylvia-from-fulp-station) [inline fulp_modules/mapping/deathmatch/code/deathmatch_loadouts.dm:1]: Top 10 commits on fulpstation repository

**Comment by Horatio22** [comment]
Maps are now in their own folder; please move this to be in with the rest and I will merge it. 

https://github.com/fulpstation/fulpstation/pull/1170

**Pull request merged**